android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt

```diff
@@ -61,7 +60,7 @@ class Peripheral(context: Context, peripheralListener: IPeripheralListener) {
 
   fun sendData(serviceUUID: UUID, charUUID: UUID, data: ByteArray) {
     val currentState = messageSender.getCurrentState()
-    if (currentState == StateHandler.States.CommunicationReady) {
+    if (currentState == StateHandler.States.CommunicationReady || (serviceUUID === GattService.DISCONNECT_CHAR_UUID)) {
```

Comment:
Why is this needed? 
Also the condition should be `charUUID === GattService.DISCONNECT_CHAR_UUID` ?

Reply:
Yes. it has to be charUUID. I will update this. 

In a case, where communication not ready yet and state is connected. And if disconnect happens, Verifier is not able to send disconnect notification. I don't remember facing this in current state. but when I tried throwing exception onConnected. this was happening.

Reply:
Fixed this in a new commit

Reply:
Fixed this in a new commit